.env.example

```diff
@@ -0,0 +1 @@
+PRIVATE_KEY=
```

Comment:
Can you add a dummy private key ? to know the format

Reply:
Added: https://github.com/AstarNetwork/wasm-showcase-dapps/pull/40/commits/dd49bb6d6e7622df130e1f1681e7887cf4848832